x-pack/test_serverless/shared/services/index.ts

```diff
@@ -5,11 +5,13 @@
  * 2.0.
  */
 
+import { SvlReportingServiceProvider } from '../../functional/services/svl_reporting';
```

Comment:
The shared services are available in both, API integration and functional UI tests and it looks like that's how you actually use that service, so it's correct to put it here. But then it would also make sense to move the implementation from `functional/services/svl_reporting` to `shared/services`.